kolibri/plugins/setup_wizard/assets/src/api.js

```diff
@@ -2,6 +2,7 @@ import { Resource } from 'kolibri.lib.apiResource';
 import pickBy from 'lodash/pickBy';
 import urls from 'kolibri.urls';
 import redirectBrowser from 'kolibri.utils.redirectBrowser';
+import clientFactory from '../../../../core/assets/src/core-app/baseClient';
```

Comment:
if we cannot use `import client from 'kolibri.client'` then `clientFactory` should probably be added to `apiSpec.js` to avoid relative imports across the plugin/core boundary

Reply:
@indirectlylit  Thank you! I didn't want to alter the kolibri JS api but it's clear this can't cause any damage, so done!

Reply:
Yeah currently `apiSpec` contains both "external API" that external plugins reference as well as "internal API" that is only used in the kolibri repo and is free to change at any time. @rtibbles and I have discussed making this distinction more explicit and also starting to version the external parts (https://github.com/learningequality/kolibri/issues/527)